### DISCUSSION TITLE
Jinja module support

### DISCUSSION THREAD

**Discussion by sdktr** [InfrastructureAsCode-ch/nettowel#12] [Q&A]
What a very nice tool you got here, looks really nice. 

My first test with the jinja command was ran against a templat with the 'ipaddress' module in it, to convert a subnetmask to CIDR notation. Nettowel errored on 'no module named ipaddress'. Where/how (if at all) is module lookup supposed to work in this project?

**Comment by ubaumann** [comment]
I assume it was an ansible template, or how did you get the custom filter? Can you share an example?
I need to think about how I can support custom filters easily.
